Posit Infrastructure/FAQs.md

```diff
@@ -63,6 +63,22 @@ install.packages("phsmethods")
 
 Some packages, e.g. `{xlsx}`, `{XLconnect}`, depend on an installation of [Java](https://en.wikipedia.org/wiki/Java_(software_platform)) in order to work.  There are no current or future plans to support Java on Posit Workbench.  Alternative packages such as `{openxlsx}` that do not rely on Java should be used instead.
 
+#### How do I install the `{ranger}` package?
+
+The default configurations in R do not allow the `{ranger}` package to compile and install successfully.  The developers of the `{ranger}` package switched the project to using C++14 in version 0.14.2 of the package.  R itself should default to C++14 from version 4.1.0 onwards (we have 4.1.2), unless the package specifies C++11.  It turns out the makevars for `{ranger}` specify C++11, even though it's meant to be compiled with C++14.  The fix is to add the line
```

Comment:
@CliveWG The reason I did not include any code to create a `~/.R` directory or `Makevars` file within was that there is a chance that these already exist, and code to create them could throw an error if they already exist, so there's a whole host of try..catch wrapping around these two commands that would need to be included.  I could do this, but I was trying to avoid the complexity, if I'm being honest.

Reply:
Ok that makes sense Terry. Ready to go then?